analyzer/src/main/kotlin/managers/utils/PubCacheReader.kt

```diff
@@ -64,14 +63,24 @@
             .find { !it.isSymbolicLink() && it.isFile && it.name == filename }
     }
 
-    fun findProjectRoot(packageInfo: JsonNode): File? {
+    fun findProjectRoot(packageInfo: JsonNode, workingDir: File): File? {
         val packageVersion = packageInfo["version"].textValueOrEmpty()
         val type = packageInfo["source"].textValueOrEmpty()
         val description = packageInfo["description"]
         val packageName = description["name"].textValueOrEmpty()
         val url = description["url"].textValueOrEmpty()
         val resolvedRef = description["resolved-ref"].textValueOrEmpty()
+        val resolvedPath = description["path"].textValueOrEmpty()
+        val isPathRelative = description["relative"].booleanValue()
 
+        if (type == "path" && resolvedPath.isNotEmpty()) {
+            // For "path" packages, the path should be the absolute resolved path of the "path" given in the description.
```

Comment:
This line is too long. That's why detekt is complaining. You can run detect locally using `./gradlew detekt`.

Reply:
done